extractor/build.gradle

```
@@ -1,11 +1,14 @@
+
+
 dependencies {
     implementation project(':timeago-parser')
 
-    implementation 'com.grack:nanojson:1.1'
+    implementation 'com.github.wb9688:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
     implementation 'org.jsoup:jsoup:1.9.2'
     implementation 'org.mozilla:rhino:1.7.7.1'
     implementation 'com.github.spotbugs:spotbugs-annotations:3.1.0'
     implementation 'org.nibor.autolink:autolink:0.8.0'
+    implementation 'org.json:json:20190722'
```

```suggestion
    implementation 'com.github.TeamNewPipe:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
    implementation 'org.jsoup:jsoup:1.9.2'
    implementation 'org.mozilla:rhino:1.7.7.1'
    implementation 'com.github.spotbugs:spotbugs-annotations:3.1.0'
    implementation 'org.nibor.autolink:autolink:0.8.0'
```

Also, don't forget to change it in `timeago-parser/build.gradle` as well, plus move the remaining code from `org.json` to `nanojson`.

---

> plus move the remaining code from `org.json` to `nanojson`.

I'm fairly sure I already did this, I must have scrambled things up somehow. I commited the commit again as b100b9873ff1e6177ad53510ad4b39dd92a78ff7.

---

I'm pretty sure you did that indeed.